Gems/PhysX/Code/Source/EditorColliderComponent.h

```diff
@@ -83,12 +84,18 @@ namespace PhysX
         EditorProxyShapeConfig() = default;
         EditorProxyShapeConfig(const Physics::ShapeConfiguration& shapeConfiguration);
 
+        // O3DE_DEPRECATION_NOTICE(GHI-14718)
+        // Initial value for m_shapeType needs to remain PhysicsAsset
+        // to support command to convert EditorColliderComponent to
+        // EditorMeshColliderComponent. This is because the lack of 'ShapeType'
+        // property in prefabs still need to mean it's using PhysicsAsset type.
```

Comment:
>I think this would read better if you change "need" to "needs".
Thanks, I'll fix that.

>Is it because nothing gets serialized to json if the property is equal to its default value
Exactly, so if you change the default value now you've lost knowing what value it had before.

>its default previously was PhysicsAsset so it has to stay like that otherwise it will deserialize old data which had the default value incorrectly?
No, its default previously was PhysicsAsset so it has to stay like that otherwise when the converter opens the prefab and reads the entity it'll think it's a box (if box is used as default) and therefore we would lose the information of knowing it was a PhysicsAsset (and therefore it needs to be converted).

Reply:
Yeah, that's what I meant. But like I say it took me a while to figure that out, so maybe the comment can be made clearer?

Reply:
Yeah, I wasn't inspired, was I. This the troubling bit 

"This is because the lack of 'ShapeType' property in prefabs still needs to mean it's using PhysicsAsset type."

What about this instead?

"This is because prefabs do not store in JSON the default values and therefore the converter would lose the ability to know if the type was PhysX Asset before to convert the component to a Editor Mesh Collider."